metricbeat/etc/beat.full.yml

```diff
@@ -17,10 +17,10 @@ metricbeat.modules:
     - cpu
 
     # System Load stats
-    - load
+    #- load
```

Comment:
Should we mention that we changed the defaults in the config file? Why is `load` disabled now by default?